target/linux/mediatek/dts/mt7981b-cmcc-rax3000m-nand-ubootmod.dts

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include "mt7981b-cmcc-rax3000m.dts"
+
+/ {
+	model = "CMCC RAX3000M NAND version (custom U-Boot layout)";
```

Comment:
`(custom U-Boot layout)` - this does not apply to the device model.

Reply:
ex.CMCC RAX3000M NAND 1027 version